openassessment/xblock/studio_mixin.py

```diff
@@ -159,7 +159,7 @@ def editor_context(self):
                 make_django_template_key(asmnt)
                 for asmnt in self.editor_assessments_order
             ],
-            'teams_feature_enabled': self.team_submissions_enabled,
+            'teams_feature_enabled': True,
```

Comment:
@0x29a Is this a testing change? If so can you remove it from the final version?

Reply:
@giovannicimolin ah, yes, sure.